templates/zerver/help/change-who-can-join-a-stream.md

```diff
@@ -0,0 +1,39 @@
+# Change who can join a stream
+Zulip allows you to create new private streams as a user with stream creation privileges.
```

Comment:
Private streams require invitation by the admin while public streams are open to everyone.